tests/model/Megatron_GPT2/run_checkpoint_test.py

```diff
@@ -216,9 +319,13 @@ def check_parity(self, base_file, test_file, r_tol):
 
 def checkpoint_suite():
     suite = unittest.TestSuite()
-    suite.addTest(GPT2CheckpointTestCase('test_mp4_gpu16_node1_with_zero1'))
-    suite.addTest(GPT2CheckpointTestCase('test_mp4_gpu16_node1_with_zero2'))
-    suite.addTest(GPT2CheckpointTestCase('test_mp4_gpu16_node1_without_zero'))
+    suite.addTest(GPT2CheckpointTestCase('test_mp2_gpu4_node1_with_zero1'))
+    suite.addTest(GPT2CheckpointTestCase('test_mp2_gpu4_node1_with_zero2'))
+    suite.addTest(GPT2CheckpointTestCase('test_mp1_gpu2_load_gpu1_node1_with_zero1'))
+    suite.addTest(GPT2CheckpointTestCase('test_mp1_gpu2_load_gpu4_node1_with_zero1'))
+    suite.addTest(GPT2CheckpointTestCase('test_mp1_gpu2_load_gpu1_node1_with_zero2'))
```

Comment:
Can we add one test case with mp=2, dp=2 loaded to mp=2, dp=4 or 1? Seems like we are not testing any cases with mp > 1 when the dp changes?

Reply:
Good point. Will do;